media/vp9-codec-string.js

```diff
@@ -0,0 +1,25 @@
+if (MediaSource.isTypeSupported('video/webm; codecs="vp9"')) {
+  log('> VP9, Profile 0, level 1 (later fields defaulted)');
+}
+
+if (MediaSource.isTypeSupported('video/webm; codecs="vp09.00.10.08"')) {
+  log('> VP9, Profile 0, level 1, bit depth 8 (later fields defaulted)');
+}
+
+if (MediaSource.isTypeSupported('video/webm; codecs="vp09.01.20.08.01"')) {
+  log('> VP9, Profile 0, level 1, bit depth 8, 4:2:0 chroma subsampling ' +
```

Comment:
Log doesn't match the string. String shows profile 1, level 2. Sorry, I made this typo in the web platform tests.